### DISCUSSION TITLE
Is there any limit on number of Projects in ArgoCD?

### DISCUSSION THREAD

**Discussion by vinodkumar-wagh** [argoproj/argo-cd#14345] [Q&A]
In our organization, we have a shared EKS infrastructure where all application teams are migrating their applications. To manage the EKS deployments, we are using ArgoCD. We have noticed that few teams require multiple ArgoCD projects for their specific needs. Therefore, we would like to determine if there are any limits in place to avoid potential issues or restrictions in the future.

**Comment by crenshaw-dev** [comment]
There is no limit on the number of projects. As with any resource, you may encounter performance issues as the number of resources goes from "hundreds" to "tens of thousands." But there's no hard limit.
